_episodes/08-working-with-sql.md

```diff
@@ -2,12 +2,18 @@
 title: Accessing SQLite Databases Using Python & Pandas
 teaching: 20
 exercises: 25
-
+questions:
+    - "Can I use Python to create and interact with SQLite databases?"
```

Comment:
Can we rephrase this so that it isn't a yes/no question? Maybe saying "How can I use Python to create and interact with SQLite databases?"